display_list/display_list_benchmarks_metal.mm

```diff
@@ -40,6 +58,7 @@ class MetalCanvasProvider : public CanvasProvider {
   std::unique_ptr<TestMetalContext> metal_context_;
   std::unique_ptr<TestMetalSurface> metal_surface_;
   std::unique_ptr<TestMetalSurface> metal_offscreen_surface_;
+  AutoreleasePool pool_;
```

Comment:
I don't recommend tying an autorelease pool to the lifetime of an object.  They instead should be created and drained within the scope of one event.

Reply:
OK, new plan is to refactor the tests so that we allocate the CanvasProvider in the test body, and create the AutoreleasePool object at the beginning of each test e.g.:

```
void runTest(BackendType backend_type) {
  AutoreleasePool pool;
  auto canvas_provider = CreateCanvasProvider(backend_type);
  // test stuff here
}
```